docs/bridge/docs/04-Routers/RFQ/API/index.md

```
@@ -60,6 +60,54 @@
+  All websocket messages follow this format:
+  ```
+  {
+    op: string,
+    content: json,
+    success: bool,
+  }
+  ```
+
+  Quote request content should have the following format:
+
+  ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)